src/components/button/Button.js

```diff
@@ -7,22 +7,36 @@ import LoadingIndicator from '../loading-indicator';
 import RadarAnimation from '../radar';
 
 type Props = {
-    /** Child components */
+    /** Child components for the button, generally localized text */
     children?: React.Node,
-    /** Adds a class to the component */
+    /** Custom class for the button */
     className?: string,
+    /** whether the button is disabled or not */
```

Comment:
Storybook is not able to get description from these comments. With the version I am using and CSF.